src/include/pipeline/hll.h

```diff
@@ -29,10 +29,12 @@
 #define HLL_DENSE_CLEAN 'D'
 #define HLL_EXPLICIT_DIRTY 'e'
 #define HLL_EXPLICIT_CLEAN 'E'
+#define HLL_UNION 'u'
 
 #define HLL_IS_SPARSE(hll) ((hll)->encoding == HLL_SPARSE_DIRTY || (hll)->encoding == HLL_SPARSE_CLEAN)
 #define HLL_IS_EXPLICIT(hll) ((hll)->encoding == HLL_EXPLICIT_DIRTY || (hll)->encoding == HLL_EXPLICIT_CLEAN)
 #define HLL_IS_DENSE(hll) ((hll)->encoding == HLL_DENSE_DIRTY || (hll)->encoding == HLL_DENSE_CLEAN)
+#define HLL_IS_UNION(hll) ((hll)->encoding == HLL_UNION)
```

Comment:
I think calling this `UNION` is a little confusing since it's basically just a different encoding. We should maybe call it something like `UNPACKED` and then reuse functions like `HLLUnion` and just throw errors if `UNPACKED` is being merged with something other than `UNPACKED`.